src/main/kotlin/com/example/springbootkotlin/SpringbootkotlinApplication.kt

```diff
@@ -1,11 +1,68 @@
 package com.example.springbootkotlin
 
+import com.example.springbootkotlin.web.dto.Food
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
+import java.util.*
 
 @SpringBootApplication
 class SpringbootkotlinApplication
 
 fun main(args: Array<String>) {
-    runApplication<SpringbootkotlinApplication>(*args)
+
+    // 1. KStack
+    class KStack<T> {
+        val list = mutableListOf<T>()
+
+        fun push(item: T) = list.add(item)
+        fun pop(): T = list.removeLast()
+        fun size(): Int = list.size
+        fun find(item: T): Boolean = list.contains(item)
+    }
+
+    // 2. 괄호 문제
+    val sc = Scanner(System.`in`)
+    val n = sc.nextInt()
```

Comment:
```suggestion
    val n = readln().toInt()
```

이런식으로 작성 가능, 단 Kotlin 1.6 이상